config/plugin.js

```diff
@@ -37,7 +37,21 @@ const { getHashiConfig } = require('./index')
  * See the test file at `./config/__tests__/index.test.js` for a more thorough example.
  */
 module.exports = function HashiConfigPlugin() {
-	const env = process.env.HASHI_ENV || 'development'
+	const env = process.env.HASHI_ENV
+		? (() => {
+				console.log(
+					'[HashiConfigPlugin] HASHI_ENV set to:',
+					process.env.HASHI_ENV
+				)
```

Comment:
nit: this log feels unnecessary. It adds three extra logs to each build, e.g.:
https://vercel.com/hashicorp/dev-portal/GvowgiEc2Sr8VuMeiyuhboSyezNT#L44-L46

Reply:
Good callout! I had looked into a `logOnce` function used else where, but it quickly felt like overkill.

I'll remove this log entirely!